### PR TITLE
Docs changes for Geo-Distributed Placement

### PR DESCRIPTION
**Pull request by deeps1991**
This patch contains the following changes:
(i) Edit the images for the Tablespaces section to use the name
'multi_region_tablespace' rather than 'us_multi_region_tablespace'
as the tablespaces span multiple countries
(ii) Add a new command to yb-admin page to document the command
modify_table_placement_info
(iii) Edit a bug in Row Level Geo Partitioning page wherein a
 DEFAULT partition is created, and then a new partition is created.
We should not list this here as in some cases we can't create a new
partition if a default partition is present.
(iv) Add details about the above limitation to the section on
Table Partitioning.

### CODE REVIEW

File: docs/content/latest/admin/yb-admin.md [hunk 1]
```diff
@@ -457,6 +457,53 @@ Waiting for compaction...
...
+yb-admin \
+    -master_addresses <master-addresses> \
+    modify_table_placement_info tableid.<tableid> <placement_info> <replication_factor> \
+    [ <placement_id> ]
+```
+
+* _master-addresses_: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* keyspace: The namespace, or name of the database or keyspace.
+* table_name: The name of the table.
+* table_id: The unique uuid associated with the table whose placement policy is being  changed.
```
Comment (lizayugabyte) [inline docs/content/latest/admin/yb-admin.md:485]: Extra space in "being  changed"

**Review by lizayugabyte — COMMENTED** [review]
LGTM

**Comment by deeps1991** [comment]
Added @rahuldesirazu and @bmatican mainly for the review of the yb-admin page. I've tagged this as WIP mainly to know if putting this command in the Table commands in yb-admin page looks ok.
Also wanted to ask if I should add any further details to the description of "placement_id"

File: docs/content/latest/explore/ysql-language-features/partitions.md [hunk 2]
```diff
@@ -99,6 +92,13 @@ CREATE TABLE order_changes_2021_01 PARTITION OF order_changes
 Partitioning ranges are inclusive at the lower ( `FROM` ) bound and exclusive at the upper ( `TO` ) bound.
 Each month range in the preceding examples includes the start of the month, but does not include the start of the following month.
 
+If you wish to create a new partition that merely contains all the rows that do not match the specified partitions,
+you can do so by adding a default partition in the following manner:
```
Comment (polarweasel) [inline docs/content/latest/explore/ysql-language-features/partitions.md:96]:
```suggestion
To create a new partition that contains only the rows that don't match the specified partitions, add a default partition as follows:
```

**Review by polarweasel — CHANGES_REQUESTED** [review]
Left some changes as suggestions you can batch up, and also a couple of other comments

File: docs/content/latest/admin/yb-admin.md [hunk 3]
```diff
@@ -457,6 +457,53 @@ Waiting for compaction...
...
+    modify_table_placement_info <keyspace> <tablename> <placement_info> <replication_factor> \
+    [ <placement_id> ]
+```
+
+or alternatively:
+
+```sh
+yb-admin \
+    -master_addresses <master-addresses> \
+    modify_table_placement_info tableid.<tableid> <placement_info> <replication_factor> \
```
[the remark this answers is not in the document]
Comment (deeps1991) [reply docs/content/latest/admin/yb-admin.md:478]: Yes, that is the syntax... Here is an example run:

./yb-admin modify_table_placement_info tableid.77d9bfeea4fe4578a6dc8bf4529cb51e aws.us-west-2.us-west-2a0 3

**Comment by deeps1991** [comment]
Thanks for the quick reviews @lizayugabyte and @polarweasel! Addressed your comments above.

**Review by rahuldesirazu — APPROVED** [review]

**Pull request merged**